src/doc_builder/commands/build.py

```diff
@@ -130,7 +130,7 @@ def build_command(args):
             print("Installing node dependencies")
             subprocess.run(
                 ["npm", "ci"],
-                stderr=subprocess.PIPE,
+                # stderr=subprocess.PIPE,
```

Comment:
If it's not used, why leave it commented out? Let's please clean up and remove it entirely.

Reply:
Done in 6adc10b5ede2281995ff04d95f3c94db2bbebc96

Reply:
Thanks!